dev/devicelab/lib/tasks/web_benchmarks.dart

```diff
@@ -188,3 +188,23 @@ Future<TaskResult> runWebBenchmark({ required bool useCanvasKit }) async {
     }
   });
 }
+
+Handler createBuildDirectoryHandler(String buildDirectoryPath) {
+  final Handler childHandler = createStaticHandler(buildDirectoryPath);
+  return (Request request) async {
+    final Response response = await childHandler(request);
+    final String? mimeType = response.mimeType;
+
+    // Provide COOP/COEP headers so that the browser loads the page as
+    // crossOriginIsolated. This will make sure that we get high-resolution
+    // timers for our benchmark measurements.
+    if (mimeType == 'text/html' || mimeType == 'text/javascript') {
```

Comment:
Should we also add this to `flutter run`?

Reply:
We'll have to address this question eventually, but I think we should really just do this for our benchmarks for now. Enabling `crossOriginIsolated` in `flutter run` could potentially break clients... we might want to do so conditionally or something. I think we should have a broader conversation about it, but it's probably out of scope of this PR. I'll have to figure this out for skwasm either way.

Reply:
I filed https://github.com/flutter/flutter/issues/127902 as a follow on